packages/web/components/Header/Logo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+interface IProps {
+  title: string;
+  src?: string;
+}
+
+const Logo: React.FC<IProps> = React.memo(({ title, src, ...rest }) => {
+  if (src) {
```

Comment:
Czytelniejszy zapis można by uzyskać stosując tylko jeden `return` i `ternary rendering`:
```
return (
  {src && <img ... />}
  {!src && <div ... />}
)

Reply:
Właśnie chciałem tego użyć ale nie byłem pewny jak to zrobić na dwa wykluczające się warunki - dzięki.